Check if alias is already registered before creating the alias

Sinius15 opened the pull request:
In commit 2d57cd6b7b7e6cd4a1ed9246d4bf1087785e5fc4 new code is added that creates a alias. When using default [Laravel TestCase](https://github.com/laravel/laravel/blob/master/tests/TestCase.php) this alias code causes the following crash:
`cannot redeclare class TwigBridge\Extension\Laravel\String`.
When adding a `class_exists` statement to check if the alias already exist, this error is fixed.

EDIT: this is a fix for issue https://github.com/rcrowe/TwigBridge/issues/229

shehi commented:
Are you 100% sure it shouldn't be something like:

```if ($this->isRunningOnPhp7()) {
    class_alias('TwigBridge\Extension\Laravel\Str', 'TwigBridge\Extension\Laravel\String');
}```

Notice exclamation mark gone.

Because this specific line was added for PHP 7 support. It should add features against new version, not against older versions. I didn't debug code in depth tho.

Sinius15 commented:
I am pretty sure the `!` is intentional. Reading the commit message of commit 2d57cd6b7b7e6cd4a1ed9246d4bf1087785e5fc4
`renaming class using reserved name, added alias for existing PHP5 users, added PHP7 to travis configuration.`

So `TwigBridge\Extension\Laravel\String` is renamed to `TwigBridge\Extension\Laravel\Str`. For non PHP7 users there is an alias.

rcrowe commented:
Taking a look at this now

rcrowe commented:
Thanks @Sinius15 :+1:

shehi commented:
@rcrowe : This bug is app breaking. Can we please have a release for it, I don't really want to change my composer.json just for this issue in order to downgrade.

barryvdh commented:
I would have rather had this in 0.8, as this isn't breaking right?

And could have removed the alias altogether in 0.9
